app/services/foreman/access_permissions.rb

```diff
@@ -393,6 +393,9 @@
     map.permission :edit_classes,   {:host_editing => [:edit_classes],
                                      :"api/v2/host_classes" => [:index, :create, :destroy]
                                     }
+    map.permission :view_params,  {:host_editing => [:view_params],
+                                     :"api/v2/parameters" => [:show]
```

Comment:
This permission's granted through *some* of the corresponding nested objects, e.g. view_domains/edit_domains.  Should it be migrated entirely, or removed from here and added to all nested objects?